modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -555,7 +565,7 @@
 	var/drained_blood = min(victim.blood_volume, HEMOPHAGE_DRAIN_AMOUNT, blood_volume_difference)
 
 	victim.blood_volume = clamp(victim.blood_volume - drained_blood, 0, BLOOD_VOLUME_MAXIMUM)
-	hemophage.blood_volume = clamp(hemophage.blood_volume + drained_blood, 0, BLOOD_VOLUME_MAXIMUM)
+	hemophage.blood_volume = clamp(hemophage.blood_volume + (drained_blood * (is_target_human_with_client ? 2 : 1)), 0, BLOOD_VOLUME_MAXIMUM)
```

Comment:
I feel like the ternary you added could be in its own variable, and using a define to say how much of a modifier it is.

Reply:
that last commit should do the trick